coordinator/history/fsstore.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package history
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/spf13/afero"
+)
+
+type fsStore struct {
+	fs *afero.Afero
+}
+
+func newPVStore(fs *afero.Afero) *fsStore {
+	return &fsStore{fs: fs}
+}
+
+func (s *fsStore) Get(key string) ([]byte, error) {
+	return s.fs.ReadFile(key)
+}
+
+func (s *fsStore) Set(key string, value []byte) error {
+	return s.fs.WriteFile(key, value, 0o644)
+}
```

Comment:
Do we need to `MkdirAll` at some point?

Reply:
Splitted this between constructor and set. Maybe I should also write some unit tests for the fsstore...